gamestate/user.go

```diff
@@ -45,20 +50,75 @@ func NewUserGateway(gfClient GFClient, loginSocket LoginSocket, gameSocket GameS
 	}
 }
 
+type GfAccountData struct {
+	Email    string
+	Password string
+	Locale   string
+	Name     string
+}
```

Comment:
This should be moved to the actions package which belongs to a higher layer in the infrastructure of noskit.